JetMETCorrections/Modules/src/classes_def.xml

```diff
@@ -0,0 +1,3 @@
+<lcgdict>
+    <class name="JME::JetResolution" class_version="0"/>
```

Comment:
is class_version needed?
I see that it's a common attribute in the CondFormats and nowhere else in CMSSW.

Perhaps ```persistent="false" ``` is  more useful.

Reply:
Hi Slava, I just copied from another classes_def.xml and I don't have an opinion about it. Will add ``persistent="false"`` and the end line soon.